### PR TITLE
Change the health check to use http looking at a json file to ensure …

### PR DESCRIPTION
**Pull request by jdotpz**
…we don't exercise the db and cache so heavily

### CODE REVIEW

**Comment by jdotpz** [comment]
Do not merge please.

**Comment by peterbe** [comment]
r+

I wish there was a better opportunity there to put in a comment and explain why we did this and what's so special about `/contribute.json`. 
If you have a good place to put a comment I would say
```
This webapp URL is minimal. Rendering that causes no database queries. It just tests a simple
django view that shows some JSON output. 
```

**Comment by peterbe** [comment]
NOTE: We decided to NOT use `/contribute.json`. 
Let's wait for this to land https://github.com/mozilla/socorro/pull/3160

**Comment by peterbe** [comment]
Ok, the endpoint to look for is: https://crash-stats.mozilla.com/healthcheck?elb=true
So I guess it needs to change to `target = "HTTP:80/healthcheck?elb=true"` there.

**Comment by peterbe** [comment]
ACTUALLY, https://bugzilla.mozilla.org/show_bug.cgi?id=1240107#c3

**Comment by peterbe** [comment]
Now we're waiting for https://github.com/mozilla/socorro/pull/3161

**Comment by peterbe** [comment]
https://github.com/mozilla/socorro/pull/3161 has now landed. But it's not in production yet.

**Comment by peterbe** [comment]
It is in production now. https://crash-stats.mozilla.com/monitoring/healthcheck/?elb=true
So the change needs to become this:
```
target = "HTTP:80/monitoring/healthcheck/?elb=true"
```

**Comment by peterbe** [comment]
This now depends on https://github.com/mozilla/socorro-infra/pull/230

File: terraform/webapp/main.tf [hunk 1]
```diff
@@ -144,7 +144,7 @@ resource "aws_elb" "elb-socorroweb" {
       healthy_threshold = 2
       unhealthy_threshold = 2
       timeout = 3
-      target = "TCP:80"
+      target = "HTTP:80/contribute.json"
```
Comment (peterbe) [inline terraform/webapp/main.tf]: This still needs to be changed to `"HTTP:80/monitoring/healthcheck/?elb=true"`

**Comment by jdotpz** [comment]
We're currently getting this error curling that locally:

[centos@stage-webapp-i-4cbe7c94 ~]$ curl http://localhost/monitoring/healthcheck/?elb=true
<h1>Bad Request (400)</h1>[centos@stage-webapp-i-4cbe7c94 ~]$

**Comment by peterbe** [comment]
Can we close this now? I think we have the elb healthcheck worked out now, no?

**Comment by milescrabill** [comment]
@peterbe according to https://github.com/mozilla/socorro-infra/blob/master/terraform/webapp/main.tf#L147 your health check is a TCP ping to port 80 - all that checks is that nginx is running! This would be good to address.

**Comment by milescrabill** [comment]
Closing in favor of https://github.com/mozilla/socorro-infra/pull/265.

**Pull request closed without merging**